protocol/app/ante/market_update_test.go

```diff
@@ -425,18 +443,117 @@ func TestValidateMarketUpdateDecorator_AnteHandle(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		{
+			name: "reject a single message with a new market that's enabled",
+			args: args{
+				msgs: []sdk.Msg{
+					&mmtypes.MsgUpsertMarkets{
+						Authority: constants.BobAccAddress.String(),
+						Markets: []mmtypes.Market{
+							testMarketWithEnabled,
+						},
+					},
+				},
+				simulate:         false,
+				marketMapMarkets: []mmtypes.Market{testMarket}, // existing mm market is disabled
+			},
+			wantErr: true,
+		},
+		{
+			name: "reject a single message with a new market that's enabled - simulate",
+			args: args{
+				msgs: []sdk.Msg{
+					&mmtypes.MsgUpsertMarkets{
+						Authority: constants.BobAccAddress.String(),
+						Markets: []mmtypes.Market{
+							testMarketWithEnabled,
+						},
+					},
+				},
+				simulate: true,
```

Comment:
should this have `marketMapMarkets: []mmtypes.Market{testMarket}, // existing mm market is disabled` to match the case above (with different `simulate` values)?